actions/scripts/create_run_hls_script.sh

```diff
@@ -114,6 +114,6 @@ config_interface -m_axi_addr64=true
 #config_rtl -reset all -reset_level low
 
 csynth_design
-export_design -format ip_catalog
+#export_design -format ip_catalog -rtl vhdl
```

Comment:
We might drop a little comment what this is doing, such that if folks review/see this they know what we tried to do here. Else remove entirely?